docs/cli/category-exporter.md

```diff
@@ -59,7 +59,7 @@ import fs from 'fs'
 
 const options = {
     apiConfig: {
-      apiUrl: 'https://api.sphere.io'
+      apiUrl: 'https://api.europe-west1.gcp.commercetools.com'
       host: 'https://auth.sphere.com'
```

Comment:
What about this one?

Reply:
prefect and thanks :), i really didnt notice it